pkg/apis/kudo/v1beta1/instance_types.go

```diff
@@ -59,6 +59,7 @@ type InstanceStatus struct {
 type AggregatedStatus struct {
 	Status         ExecutionStatus `json:"status,omitempty"`
 	ActivePlanName string          `json:"activePlanName,omitempty"`
+	LastUpdated    *metav1.Time    `json:"lastUpdated,omitempty"`
```

Comment:
Questions: 
- why is `LastUpdate` in the `AggregatedStatus`? We already have a `LastFinishedRun` per individual plan, wouldn't this field be similar?
- there is value in knowing when was the last time a particular plan ran, but having two timestamps, one aggregated and one per plan seems confusing to me. What if we would rename `LastFinishedRun -> LastUpdated` (and change the semantics) and whether it is finished or not would be visible from the _PlanStatus_ itself?

Reply:
In my mind, this made sense because we were showing a timestamp for when anything changed (`LastUpdated` as part of `AggregatedStatus`), and another which shows when an individual plan ran to completion, `LastFinishedRun`.  From a human operator's POV, having both helps with understanding what's happening / happened.

However, thinking on it some more, maybe the only thing we need is a single field, `LastUpdated`, as part of `PlanStatus`, which is updated any time something happens with a Plan.  This would replace `LastFinishedRun`.

I'm not sure of the history or original intent of `LastFinishedRun` however and so don't want to replace it without understanding that first.

Reply:
> I'm not sure of the history or original intent of `LastFinishedRun` however and so don't want to replace it without understanding that first.

I believe the original intent was to simply show the last finished plan in the `plan status` output so I'd be in favor of replacing it with a broader `LastUpdated`field. /cc @alenkacz wdyt?

Reply:
Yeah, we could probably get away with `LastUpdated` instead of `LastFinishedRun`. It will be a breaking change if we rename the field, but we're going to have breaking changes in the next release anyway...